.github/workflows/fuzz.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   fuzz:
     if: ${{ !github.event.act }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
maybe personal taste but what do you think about updating to a fixed version `ubuntu-22.04` ?

Reply:
We already use `ubuntu-latest` in our [main CI](https://github.com/BlockstreamResearch/rust-simplicity/blob/master/.github/workflows/main.yml#L8) so I would do the same for our fuzz CI.

[rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/.github/workflows/rust.yml#L13), [rust-secp](https://github.com/rust-bitcoin/rust-secp256k1/blob/master/.github/workflows/rust.yml#L8) and [rust-elements](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/.github/workflows/rust.yml#L13) seem to be doing the same.